lib/assessment.rb

```diff
@@ -13,28 +13,30 @@ def longest(enum)
   # enumerable supplied (i.e. apply the `+` operator)
   # Assume each entry implements a `+` method
   def sum(enum)
-    # Your code goes here
+    enum.reduce { |sum, num| sum += num }
   end
 
   # Question 3
   # Write ruby code that returns true if some of the entries
   # in the numbers enumerable are odd
   def some_odd(numbers)
-    # Your code goes here
+    numbers.any? { |num| num % 2 == 1 }
```

Comment:
Correct, but Ruby has a handy method called `.odd?` that could have made your life a bit easier.